### PR TITLE
[MRG] EHN possibility to test several submission

### PR DESCRIPTION
**Pull request by glemaitre**
`"*"` can be provided and all the submissions can be tested. We can also provide a list.

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
@glemaitre you will need to rebase as I fixed the `make test` in another PR (and switched to pytest).
 
Maybe also leave the README changes for another PR? As this will conflict with https://github.com/paris-saclay-cds/ramp-workflow/pull/35

File: rampwf/utils/command_line.py [hunk 1]
```diff
@@ -32,6 +35,16 @@ def create_parser():
 def ramp_test_submission():
     parser = create_parser()
     args = parser.parse_args()
-    assert_submission(ramp_kit_dir=args.ramp_kit_dir,
-                      ramp_data_dir=args.ramp_data_dir,
-                      submission_name=args.submission)
+
+    if args.submission == "ALL":
```
Comment (jorisvandenbossche) [inline rampwf/utils/command_line.py]: Can you add this feature to do description of the arg?

**Comment by glemaitre** [comment]
I'll do that when I get out of this bus...

**Comment by glemaitre** [comment]
I corrected `.travis.yml` which doest not need to fetch the ramp-kits anymore since it is done in the testing of the rampwf itself.

**Comment by glemaitre** [comment]
@jorisvandenbossche Changes done

**Pull request merged**